pkg/trace/api/telemetry.go

```diff
@@ -0,0 +1,11 @@
+package api
+
+var telemetryProxyEndpointsConfig = proxyEndpointsConfig{
+	name:                      "telemetry",
+	enabledConfig:             "apm_config.telemetry.enabled",
+	ddURLConfig:               "apm_config.telemetry.dd_url",
+	additionalEndpointsConfig: "apm_config.telemetry.additional_endpoints",
```

Comment:
```suggestion
	name:                      "telemetry",
```
What if we keep just `telemetry`? The rest can be deduced from it using `apm_config.<name>.{enabled,dd_url,additional_endpoints}`